### DISCUSSION TITLE
Introducing a new plugin - Where? How?

### DISCUSSION THREAD

**Discussion by imfx77** [kanboard/kanboard#5417] [General]
Hi all,
I have just published a pre-release of my very first KB plugin, one I've been developing for the last month.
[Wysiwyg MD Editor plugin for Kanboard](https://github.com/imfx77/kanboard-plugin-Wysiwyg-MD-Editor)

And I am kinda confused hot to announce it to the community, where and how to enlist it, how to push it to undergo some user testing and reviewing ...

I am mostly targeting the question to both of you @fguillot and @aljawaid, since the first is the major creator and maintainer of KB, and the second has lately been involved in organizing and listing plugins in the **PluginManager**.

I have found the following resources, yet they seem not particularly useful to me nor answering my request:
- I guess [Kanboard Plugins](https://kanboard.org/plugins.html) is the official list but there seems to be no way to enlist my plugin and there's no indication whatsoever if it is actually being updated and how
- Some discourse group threads like [this](https://kanboard.discourse.group/t/new-plugin-pluginmanager/2492) and [that](https://kanboard.discourse.group/t/how-can-we-improve-kanboard-plugins-information/2325) make me believe that the matter has been discussed numerous times to no avail ... so probably no point posting there ...
- Your very private chat here [Can we Update the Plugins Website?](https://github.com/orgs/kanboard/discussions/5185) seems to be latent for 2+ years and gives me no hints
- And finally [kanboard / website](https://github.com/kanboard/website) suggests that I add myself the plugin to the site by editing the `plugins.json` and `plugins.html` files - which means I have to checkout the repo, mangle the files, send a PR request and wait for it to be merged ... and eventually appear in the official list on [Kanboard Plugins](https://kanboard.org/plugins.html) ...

So ...
Is this the official way? The only way? How long does it take to get **approved**?
Or should I just shout around the link to my repo in various media, so to make people aware of the new plugin?
How do I announce a pre-release, so to get some preliminary feedback?

Regards,
Ilian

**Comment by aljawaid** [comment]
Hi

> And finally kanboard / website suggests that I add myself the plugin to the site by editing the plugins.json and plugins.html files - which means I have to checkout the repo, mangle the files, send a PR request and wait for it to be merged ... and eventually appear in the official list on Kanboard Plugins ...

- this is the official way
